geps/gep-1651.md

```diff
@@ -0,0 +1,408 @@
+# GEP-1651: Gateway Routability
```

Comment:
Please add this to mkdocs.yml (https://github.com/kubernetes-sigs/gateway-api/blob/main/mkdocs.yml#L81)